.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "admu_student_app",
+            "request": "launch",
+            "type": "dart"
+        }
+    ]
+}
```

Comment:
add in `.vscode`  in `.gitignore`  so that this folder and its sub files won't be uploaded in GitHub

This are unnecessary